features/p2/repo/pom.xml

```diff
@@ -201,6 +201,11 @@
                   <transitive>false</transitive>
                   <source>true</source>
                 </artifact>
+                <artifact>
+                  <id>org.openhab.core.bundles:org.openhab.core.io.bin2json:${project.version}</id>
+                  <transitive>false</transitive>
```

Comment:
In the openhab-core BOM you inserted it keeping the alphabetical order. Would it be possible to move it to the "right" place here, too?